dna/course/zomes/courses/code/src/section/anchor.rs

```diff
@@ -0,0 +1,62 @@
+#[derive(Serialize, Deserialize, Debug,self::DefaultJson, Clone)]
+pub struct SectionAnchor {
```

Comment:
similarly to Section entry, we need to have `course_anchor_address` here too, to implement the implicit link from section to course: it will also come in handy when implementing handlers for Section